_data/accessibility-tests/typography.yml

```diff
@@ -0,0 +1,32 @@
+title: Typography
+component_status: pass
+test_items:
+# General tests
+  - summary: Text meets color contrast requirements.
+    summary_additional: |
+      When you view text on a page and use ANDI or color contrast analyzer to look at the hex colors,
+      the font color meets WCAG color contrast of 4.5:1.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.6.1
+    wcag_criterion: 1.4.3
+  - summary: Headings are present, appropriate and follow logical order.
+    summary_additional: |
+      When you view text content on a page,
+      headings summarize the content under them and are uniquely styled.
+      `H1`-`H4` levels are nested in a logical order (e.g., levels don't get skipped).
```

Comment:
Was wondering the same thing. Does USWDS only use H1-H4? If we do, it could make sense to keep it that way. However, I think we should include smaller headings like H5's. Could we do something like, "H1, H2, etc." to show an example?

Reply:
I've heard/read for a long time that most pages should not go below an H4 (they CAN but SHOULD they?). It's possible that's why we only show levels from 1-4. The reasons being are:
The pages get too long and unwieldy
H5 and H6 are often unstyled by designers
They can be misused as decorative headings and not semantic ones

That said, since they are TECHNICALLY OK to use, we should probably include H5-H6 in the guidance. If USWDS does have styled headings below an H4 we should follow our own example and use them.

Reply:
I've heard the same thing. However, if we are using them to some extent, I think it makes sense to follow our own style as well. And then add in some guidance later on discussing if you plan to us smaller headings, ensure they are styled and coded correctly.

Reply:
USWDS does provide styles for h1-h6, so I have updated the description to include through h6 in [b6cfad9](https://github.com/uswds/uswds-site/pull/2540/commits/b6cfad99eed24c85844cc44e27240e3afcc44bdf)